src/services/VaultService/ethereum.ts

```diff
@@ -18,13 +21,17 @@ import {
 
 // TODO: refactor to use SDK instead of utils and move utils common mappings to service
 export default class EthereumService implements VaultService {
```

Comment:
This file is modified to encapsulate StrategyReport logic to pull strat reports via Subgraph. This previously occurred in a random util, but is now apart of the VaultService interface.